packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -285,14 +285,13 @@ describe('ReactTestUtils.act()', () => {
         return ctr;
       }
       const el = document.createElement('div');
+      act(() => {
```

Comment:
Why did you move this to a separate `act`? I would expect moving the assertion outside `act` is sufficient.

Reply:
the effect wouldn't flush effects until the act exits, only after which the timeout in the component would start, failing the assertion. hence, 2 act()s.

Reply:
Why did you remove the `await` then?